### PR TITLE
Feature/jo acceptance tests

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Team-Services-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Added basic landing pages for website and Eula.
- Added Unit tests
- Added Acceptance tests

@ALM-Rangers/Team-Services-Bot

### CODE REVIEW

File: src/Common/TestCodeRules.ruleset [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
     <Rule Id="CA1014" Action="None" />
     <Rule Id="CA1020" Action="None" />
+    <Rule Id="CA1062" Action="None" />
```
Comment (Hussar77) [inline src/Common/TestCodeRules.ruleset]: Why we can't validate public method parameters?

File: src/Common/TestCodeRules.ruleset [hunk 2]
```diff
@@ -4,6 +4,8 @@
...
+    <Rule Id="CA1303" Action="None" />
```
Comment (Hussar77) [inline src/Common/TestCodeRules.ruleset]: Please set **Localizable(false)** on those parameters which require no localization rather then disabling the rule.

File: src/Team-Services-Bot.Api/Controllers/DefaultController.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+// ———————————————————————————————
+
+namespace Vsar.TSBot
+{
+    using System.Web.Mvc;
+
+    /// <summary>
+    /// Provides the controller logic for the default pages.
+    /// </summary>
+    public class DefaultController : Controller
```
Comment (Hussar77) [inline src/Team-Services-Bot.Api/Controllers/DefaultController.cs]: There is no unit test for DefaultController.

File: src/Common/TestCodeRules.ruleset (hunk 1 above)
Comment (jeffrey-opdam) [inline src/Common/TestCodeRules.ruleset]: These are rules for the tests (acceptance tests, unit tests). I am not very keen to do it there.

File: src/Common/TestCodeRules.ruleset (hunk 2 above)
Comment (jeffrey-opdam) [inline src/Common/TestCodeRules.ruleset]: Same applies, these are rules for the tests (acceptance tests, unit tests). I am not very keen to do it there.

File: src/Team-Services-Bot.Api/Controllers/DefaultController.cs (hunk 3 above)
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api/Controllers/DefaultController.cs]: True, I will get them done.

**Comment by jeffrey-opdam** [comment]
I added the unit tests for default controller.

**Pull request merged**